lxa_iobus/lpc11xxcanisp/can_isp.py

```diff
@@ -384,8 +385,9 @@ def flash_image(self, start, data):
             logging.info("Send block %d", block_num)
 
             start_offset = block_size * (block_num - start_sector)
+            end_offset = start_offset + block_size
 
-            block = data[start_offset : start_offset + block_size]
+            block = data[start_offset:end_offset]
```

Comment:
Same here. I am not a fan of all these subtle changes we are doing here.
Are these changes done to quiet the battle between black and flake on how to write these lines? If so please just quiet the flake-warning on that line and reduce the amount of code changes!

Reply:
How about having these lines `# noqa`ed out in the initial commit that does the other flake8 fixes and having a followup commit that changes the range indexing to the way it is now?
I would prefer not having `# noqa` everywhere.

Reply:
I would prefer a `# noqa E203` on the lines affected by the ambiguity over the workarounds.
I am quite sure that I would miss an error in those changes during review. And we do not have tests to cover these changes either.